ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -390,9 +392,12 @@ private ExecutorService getTimeoutHandlingExecutor() {
    * @throws InterruptedException is thrown when the process is interrupted because of timeout for example
    */
   @VisibleForTesting
-  protected Boolean findNextCompactionAndExecute(boolean computeStats) throws InterruptedException {
+  protected Boolean findNextCompactionAndExecute(boolean computeStats) {
```

Comment:
please fix the javadoc also

Reply:
fixed

Reply:
btw InterruptedException is thrown from checkInterrupt

Reply:
but it's handled inside of findNextCompactionAndExecute